LOAN/LoansSpecific/GreenLoans.rdf

```diff
@@ -108,6 +121,90 @@
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-dt-fd;hasSchedule"/>
+				<owl:onClass rdf:resource="&fibo-fnd-agr-ctr;MilestoneSchedule"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-agr-ctr;hasContractualElement"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-agr-ctr;DisclosureProvision"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-gao-obj;hasStrategy"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-spc-grn;SustainabilityAndBusinessStrategy"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">sustainability-linked loan</rdfs:label>
+		<skos:definition xml:lang="en">credit agreement and/or contingent facility (such as a bonding lines, guarantee line or letter of credit) for which the economic characteristics can vary depending on whether the borrower achieves ambitious, material and quantifiable predetermined sustainability performance objectives aligned with Sustainability-Linked Loan Principles (SSLP)</skos:definition>
+		<skos:example xml:lang="en">The use of proceeds in relation to a SLL is not a determinant in its categorisation and, in most instances, SLLs will be used for general corporate purposes. Instead, SLLs look to support a borrower in improving its sustainability performance.</skos:example>
+		<cmns-av:abbreviation xml:lang="en">SLL</cmns-av:abbreviation>
+		<cmns-av:adaptedFrom xml:lang="en">https://www.addleshawgoddard.com/en/insights/insights-briefings/2020/financial-services/green-loans-and-sustainability-linked-loans-what-is-the-difference/</cmns-av:adaptedFrom>
+		<cmns-av:adaptedFrom xml:lang="en">https://www.lsta.org/content/sustainable-lending-glossary-of-terms/</cmns-av:adaptedFrom>
+		<cmns-av:explanatoryNote xml:lang="en">A borrower&apos;s sustainability performance is measured using sustainability performance targets (SPTs), which include key performance indicators, external ratings and/or equivalent metrics that measure improvements in the borrower&apos;s sustainability profile.</cmns-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-spc-grn;SustainabilityPerformanceTarget">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-utl-alx;QualifiedMeasure"/>
+		<rdfs:label xml:lang="en">sustainability performance target</rdfs:label>
+		<skos:definition xml:lang="en">quantitative measure used to calibrate the level of achievement a borrower makes with respect to a key performance indicator, including, but not limited to, the methodology used to calculate its value at any point over the lifetime of a loan</skos:definition>
```

Comment:
I was not able to see any logical/conceptual link to https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/Analytics/KeyPerformanceIndicator.

Reply:
done